### DISCUSSION TITLE
Indicate whether to send a cookie in a cross-site request by specifying its SameSite attribute

### DISCUSSION THREAD

**Discussion by hasi94** [community/community#27995] [Other Feature Feedback, Questions, & Ideas]
Error:
Getting the error "Indicate whether to send a cookie in a cross-site request by specifying its SameSite attribute" on my Laravel application.
This is ONLY happening in Chrome and Edge. Works fine on Firefox.
Below is the configuration I have in session.php

`'path' => '/',
'secure' => env('SESSION_SECURE_COOKIE', true),
'http_only' => true,
 'same_site' => 'none',`

**Comment by 0xNN** [comment]
Just checked a project where the site is loaded in an iframe. The critical parts are

secure = true
same_site = none
http_only = false
